packages/lesswrong/components/posts/PostsPage/PostsPageEventData.tsx

```diff
@@ -35,19 +56,30 @@ const styles = (theme: ThemeType): JssStyles => ({
     fontSize: 15,
     marginTop: -4,
     marginLeft: 6
-  }
+  },
+  mapbox: {
+    flex: 'none',
+    width: 300,
+    [theme.breakpoints.down('xs')]: {
+      marginTop: 20
+    },
+  },
 })
 
 const PostsPageEventData = ({classes, post}: {
   classes: ClassesType,
-  post: PostsBase,
+  post: PostsList,
 }) => {
   const {captureEvent} = useTracking()
   
-  const { location, contactInfo, onlineEvent, eventRegistrationLink, joinEventLink } = post
+  const { location, contactInfo, onlineEvent, eventRegistrationLink, joinEventLink, eventType } = post
   
   // event location - for online events, attempt to show the meeting link
-  let locationNode = location && <div>{location}</div>
+  let locationNode = location && <div>
+    <a href={`https://www.google.com/maps/search/?api=1&query=${encodeURIComponent(location)}`}>
+      {location}
+    </a>
```

Comment:
I saw this on another site and immediately wanted this feature - just a link to the location on Google Maps.